### ISSUE TITLE
Highlight color changed in 0.26

### ISSUE THREAD

**Issue by marnen** [kevinsawicki/monokai#109]
I just upgraded to Monokai 0.26, and the highlight color used for files in the tree view, tabs, and checkboxes changed from blue to green. (I wouldn't expect a syntax theme to affect this at all, but apparently it does.) Which color variable is involved here? I've tried inspecting the elements in question, but I can't really tell...

**Comment by gustavoalfaro** [comment]
Do you have a screenshot?

**Comment by marnen** [comment]
![image](https://user-images.githubusercontent.com/34378/98414108-c7fe5480-2048-11eb-9a8f-1f2bbd0e21e5.png)
![image](https://user-images.githubusercontent.com/34378/98414118-cfbdf900-2048-11eb-830d-86ecdf1993dd.png)
![image](https://user-images.githubusercontent.com/34378/98414147-d9476100-2048-11eb-956c-90385bc3f47d.png)

All the blue elements in these images changed to green when I upgraded.

**Comment by gustavoalfaro** [comment]
It's still green for me:
![image](https://user-images.githubusercontent.com/21284308/98418832-625e9800-2049-11eb-8204-ad5fd06c7696.png)

**Comment by marnen** [comment]
@gustavoalfaro Yes, your green screenshots look like what I get in 0.26, and why I opened this issue. The blue screenshots are from another installation that’s running an older version of the same theme.

**Comment by marnen** [comment]
So, to recap: when I upgraded from 0.24 to 0.26, the highlight color changed from blue (0.24) to green (0.26). How do I change it to blue? I didn’t see an obvious LESS variable.

**Comment by sundaycrunk** [comment]
I've got the same issue. I went through every file in the monokai folder, and tried changing every possible color that could be resulting in this green highlight and background overlay/underlay and it's still there...

![Screen Shot 2020-11-07 at 1 21 39 pm](https://user-images.githubusercontent.com/68580909/98429725-89c26d00-20fc-11eb-97fa-04f5cd52cf21.png)

**Comment by caleb531** [comment]
@marnen This is because, if you look at the diff between Monokai v0.24.0 and v0.26.0:

```sh
git diff v0.24.0..v0.26.0
```

You will see that the `@syntax-background-color` changed in `syntax-variables.less`:

```diff
- @syntax-background-color: @dark-gray;
+ @syntax-background-color: @sundried-clay;
```

And, by consequence, the one-dark-ui theme dynamically sets the focus color (i.e. green vs. blue) based on the saturation of the @syntax-background-color:

![Screen Shot 2020-11-08 at 9 38 56 AM](https://user-images.githubusercontent.com/872474/98472201-62c07580-21a6-11eb-9f8e-d129bd677fdc.png)

However, you can form the `monokai` theme and change the `@syntax-background-color` to `@dark-gray` in `syntax-variables.less`, then relink it to your local packages. I've already taken the liberty of creating the patched fork for you:

```
git clone https://github.com/caleb531/monokai.git
cd monokai
rm -rf ~/.atom/packages/monokai
apm link
```

Restart Atom, and you're done!

**Comment by marnen** [comment]
@caleb531 Huh, interesting. I did see the change from `@dark-gray` to `@sundried-clay`, but it never would have occurred to me that that changed the highlight color from blue to green!

Thanks for the patch, too; I'll look at it. I wonder if this should be considered a bug in One Dark...

**Comment by gustavoalfaro** [comment]
Ok makes sense, I'm going to release a fix for this!

**Comment by Nerd2k** [comment]
@gustavoalfaro what was the reason for the change from @dark-gray to @sundried-clay in the first place?
I personally liked it better the way it looked with @dark-gray.

**Comment by gustavoalfaro** [comment]
I think that was a change from @ppete2 who was trying to make it more similar to the original theme from Sublime Text but I also like the `@dark-gray` since is more comfortable for the eyes.

**Comment by Nerd2k** [comment]
> I think that was a change from @ppete2 who was trying to make it more similar to the original theme from Sublime Text

Ok, I see.
> but I also like the `@dark-gray` since is more comfortable for the eyes.

Yes, same for me.

**Comment by gustavoalfaro** [comment]
Fix released in v0.27
